### DISCUSSION TITLE
Dmx mapping

### DISCUSSION THREAD

**Discussion by surajchauhan6631** [xLightsSequencer/xLights#4561] [General]
I’m trying to find an efficient way to send RGB data from an image to a DMX lighting set up. For example, say you had 20 light fixtures, I want to position these on my image and source the RGB colour data for the pixels in that location, so I can send out via DMX to the lights.

**Comment by AzGilrock** [comment]
Kinda depends on what type of DMX model it is.  For simple 3 channel flood light type models just put all the models in a group and use a Per Preview render buffer and drop a Pictures or Video effect on the group.  Other more complex DMX models require a render buffer that aligns the channels in a single line so for a group you would probably need to create a shadow model that overlays the channels you need to control.

**Reply by surajchauhan6631** [reply]
Yes it's just simple 3 channel RGB model
